src/presentation/web/client/index/index.css

```
@@ -214,6 +214,247 @@ body.has-player .source-panel {
+  height: 2.45rem;
+  min-height: 2.45rem;
+  padding: 0;
+  justify-self: stretch;
+}
+
+.download-action.icon-only::before {
+  content: "";
+  display: block;
+  width: 1.05rem;
+  height: 1.05rem;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)